.github/workflows/ci2.yml

```diff
@@ -68,21 +68,18 @@ jobs:
           - distro: rocky
             base: centos.8
             with: rockylinux/rockylinux:8
-            java: yes
           - distro: fedora
             base: centos.8
             with: fedora:35
-            java: no
           - distro: leap.15
             base: leap.15
             with: leap.15
-            java: yes
     env:
       DEPS_JOBS: 10
       BASE_DISTRO: ${{ matrix.with }}
       DOCKER_BASE: ${{ matrix.base }}
       COMPILER: ${{ matrix.compiler }}
-      DAOS_JAVA_BUILD: ${{ matrix.java }}
+      DAOS_JAVA_BUILD: yes
```

Comment:
As this is set on L111 you don't need it in the env now.

Reply:
> did you see my earlier comment about needing to update the landings triggers as well?

I'll do it probably tomorrow.